lib/cf/broker/src/auth/uaa.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const request = require('abacus-request');
```

Comment:
We should use the pattern from here: https://github.com/cloudfoundry-incubator/cf-abacus/blob/6e6a18a80e0810147b968fb88e41faef0bc075f8/lib/cf/renewer/src/index.js#L27-L37

This guarantees us retries, prevents DoS and OOM due to exhausted connections